webapp/platform/client/src/websocket.test.ts

```diff
@@ -209,4 +209,131 @@ describe('websocketclient', () => {
 
         jest.useRealTimers();
     });
+
```

Comment:
0/5 if it makes sense, but would be great to have a test around the callback not being called after a normal close. The high level way I would do it would be:

- Open the websocket connection
- Wait for the first ping to be made (with the fake timers) to make sure the interval is running
- Close the connection
- Wait for the amount of time that the interval needs (again with fake timers)
- Verify nothing related to the ping has been called